qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -651,6 +652,11 @@ def is_recently_audible(self):
         page = self._widget.page()
         return page.recentlyAudible()
 
+    def _on_url_changed(self):
```

Comment:
This should be decorated with `@pyqtSlot()` (or `@pyqtSlot(QUrl)` with the comment below).